ScanUnsupportedChars/ScanUnsupportedChars.ps1

```diff
@@ -72,13 +80,18 @@
       .\ScanUnsupportedChars.ps1  -SharePath  \\server\SyncShare  -CsvPath "C:\temp"
       This would scan unsupported file names for the provided remote SyncShare and puts the output in the
       CSV formatted files in the output directory.
+ 
+ .EXAMPLE
+      .\ScanUnsupportedChars.ps1  -SharePath  \\server\SyncShare  -DestinationPath "\\some\server" -RoboCopyOptions '/SEC'
+      This would scan unsupported file names for the provided remote SyncShare and puts the output in the Destination Path
 
  .NOTES
      Script Limitations:
      1. If file name have only the unsupported chars in the name and provided ReplacementString is empty string,
         the fixed file name would be empty string, hence cannot be renamed into.
      2. If after replacing the unsupported chars, two or more files gets the same fixed name, the script would not
         be able to rename them due to name collision.
+     3. If Folders have unsupported chars in the name, Azure FileSync will not pick them up, but folder names are not yet scanned
```

Comment:
This is confusing. AFS wont sync the folder and the files under it.

Reply:
Yes - as handling such folders was not part of initial scope, I wanted to mention this as limitation of the script. But as this case is now handled as well I removed the whole mentioning of the limitation again.